kiali/README.md

```diff
@@ -65,18 +81,30 @@ If you use [Jaeger](https://www.jaegertracing.io/) for distributed tracing, Kial
 
 For integration instructions, read [Kiali: Jaeger configuration](https://kiali.io/docs/configuration/p8s-jaeger-grafana/jaeger/).
 
+To integrate with the Kyma tracing component run:
+
+```bash
+export JAEGER_URL=`kubectl -n kyma-system get virtualservices.networking.istio.io tracing -ojsonpath='{.spec.hosts[0]}'`
+helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.external_services.tracing.enabled=true --set cr.spec.external_services.tracing.url=https://${JAEGER_URL} --set cr.spec.external_services.tracing.in_cluster_url=http://tracing-jaeger-query.kyma-system:16686 --set cr.spec.external_services.tracing.use_grpc=false -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+```
+
+Ensure that access to the Jaeger query service from the Kiali pod is allowed by the active Istio Authorization Policies. For the Kyma tracing component, use the override values in [tracing-values.yaml](tracing-values.yaml) and adapt the namespace for the used principal.
```

Comment:
```suggestion
Access to the Jaeger query service from the Kiali Pod must be allowed by the active Istio Authorization Policies. 
For the Kyma Tracing component, use the override values in [tracing-values.yaml](tracing-values.yaml) and adapt the Namespace for the used principal.
```

Does this have to happen before or after connecting with previous bash command? If before, why do we write it afterwards?

Reply:
The order is not important